TESTING.asciidoc

```diff
@@ -221,30 +221,40 @@ mvn test -Dtests.jvm.argline="-XX:HeapDumpPath=/path/to/heapdumps"
 
 Running backwards compatibility tests is disabled by default since it
 requires a release version of elasticsearch to be present on the test system.
-To run backwards compatibilty tests untar or unzip a release and run the tests
-with the following command:
 
+Until there is a GA release in the 2.0 line follow these temporary instructions
+to setup the bwc test environment execute the following steps:
+
+
+1. Go to
+https://oss.sonatype.org/content/repositories/snapshots/org/elasticsearch/distribution/tar/elasticsearch/2.0.0-beta2-SNAPSHOT/
```

Comment:
we can use rc1 now, but I certainly hope that by the time this PR gets, GA is released